key.core/src/main/java/de/uka/ilkd/key/nparser/ParsingFacade.java

```diff
@@ -128,21 +129,22 @@ public static KeyAst.File parseFile(File file) throws IOException {
 
     public static KeyAst.File parseFile(CharStream stream) {
         KeYParser p = createParser(stream);
-
+        var ep = p.getErrorReporter();
         p.getInterpreter().setPredictionMode(PredictionMode.SLL);
         // we don't want error messages or recovery during first try
-        p.removeErrorListeners();
-        p.setErrorHandler(new BailErrorStrategy());
```

Comment:
Usually this BailOutStrategy is used to avoid that ANTLR tries to repair. Why is it better to remove it?

Reply:
I have integrated the bailout strategy into the `SyntaxErrorReporter`. Now every keyparser does not try to recover. This should solve skipped definitions.